src/app.tsx

```diff
@@ -183,63 +197,56 @@ export const Application = () => {
     const removeAlert = (key: string) => setAlerts(prevAlerts => prevAlerts.filter(alert => alert.key !== key));
 
     return (
-        <Page className="pf-m-no-sidebar" isContentFilled>
-            <FilesContext.Provider value={{ addAlert, removeAlert, cwdInfo }}>
+        <FilesContext.Provider value={{ addAlert, removeAlert, cwdInfo }}>
+            <UploadContext.Provider value={{ uploadedFiles, setUploadedFiles }}>
                 <WithDialogs>
-                    <AlertGroup isToast isLiveRegion>
-                        {alerts.map(alert => (
-                            <Alert
-                              variant={alert.variant}
-                              title={alert.title}
-                              actionClose={
-                                  <AlertActionCloseButton
-                                    title={alert.title}
-                                    variantLabel={`${alert.variant} alert`}
-                                    onClose={() => removeAlert(alert.key)}
-                                  />
-                              }
-                              actionLinks={alert.actionLinks}
-                              key={alert.key}
-                            >
-                                {alert.detail}
-                            </Alert>
-                        ))}
-                    </AlertGroup>
-                    <FilesBreadcrumbs path={path} />
-                    <PageSection
-                      hasBodyWrapper={false}
-                      id="files-folder-section"
-                      data-dir-loaded={!loadingFiles ? path : null}
-                    >
+                    <Page className="pf-m-no-sidebar" isContentFilled>
+                        <AlertGroup isToast isLiveRegion>
+                            {alerts.map(alert => (
+                                <Alert
+                                  variant={alert.variant}
+                                  title={alert.title}
+                                  actionClose={
+                                      <AlertActionCloseButton
+                                        title={alert.title}
+                                        variantLabel={`${alert.variant} alert`}
+                                        onClose={() => removeAlert(alert.key)}
+                                      />
+                                  }
+                                  actionLinks={alert.actionLinks}
+                                  key={alert.key}
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-1237-7181d291-20251114-151205-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)